data/tutorials/getting-started/1_01_a_tour_of_ocaml.md

```diff
@@ -150,14 +150,14 @@ When `square` is applied to `50`, it evaluates `x * x` into `50 * 50`, which lea
 The REPL indicates that the type of `square` is `int -> int`. This means it is a function taking an `int` as a parameter (input) and returning an `int` as result (output). A function value can't be displayed, which is why `<fun>` is printed instead.
 
 ```ocaml
-# let square ~x = x * x;;
-val square : x:int -> int = <fun>
+# let sq ~x = x * x;;
+val sq : x:int -> int = <fun>
 
-# square ~x:50;;
+# sq ~x:50;;
 - : int = 2500
 ```
 
-This example defines `square` again but it has a named parameter this time. Function parameters can have labels. Note that the label is included when applying `square`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial.
+This example defines the square function again but it has a named parameter this time. Function parameters can have labels. Note that the label is included when applying `sq`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial. Labels are the most useful when a function has several parameters and being able to name the arguments makes the code easier to understand.
```

Comment:
```suggestion
This example defines the square function again but it has a named parameter this time. Function parameters can have labels. Note that the label is included when applying `sq`. Labelled parameters are detailed in the [Labelled Arguments](/docs/labels) tutorial. Labels are the most useful when a function has several parameters. The ability to name the arguments makes the code easier to understand.
```

Reply:
I would explicitely say somewhere "when function parameters have the same type" but maybe that's too advanced :-)

The rationale for this is that people usually use the types to guess what the function is doing - but when 2 parameters have the same type, that guess becomes impossible. On the other hand, when all the parameters have a different type, it's usually enough to understand what the function is doing, and in this case, labels could sometimes bring a bit too much noise. But again, maybe that's just too advanced for this tutorial - in that case feel free to keep the level of details you think is best :-)

Reply:
In “tour of ocaml”, we probably need an example which only involves passing labelled arguments rather than defining a function with labelled parameters. Remember, in this one, we give the smallest taste of whatever's shown.

Here, the idea is something like:
> If you write `String.is_prefix "state" "stateless"` you'll get a warning. You should write `String.is_prefix ~prefix:"state" "stateless"`. 

And refer to the labelled arguments tutorial for more information.

That said, what you suggest is missing in the tutorial dedicated to labelled arguments. We definitely need to add it there.

Reply:
Suggestion became outdated after https://github.com/ocaml/ocaml.org/pull/1951/commits/6c387806417202b80c6754feb7eb0aa32c33f22f